homeassistant/components/esphome/quality_scale.yaml

```diff
@@ -0,0 +1,73 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: Actions are per device and not per integration.
```

Comment:
What kind of services are these?

Reply:
These are user defined actions configured in the ESPHome YAML
https://esphome.io/components/api.html#user-defined-actions

Reply:
clarified the exemption in 8e275635cd6